lib/java/desktop-environment/src/main/java/org/enso/desktopenvironment/directories/DirectoriesFactory.java

```diff
@@ -0,0 +1,30 @@
+package org.enso.desktopenvironment.directories;
+
+import org.enso.desktopenvironment.Platform;
+
+public final class DirectoriesFactory {
```

Comment:
Please simplify. No _factory_, unless really needed. In my opinion a getter in `Platform` is a good way to obtain `Directories` instance. No need to expose _factory_ to public.